static/js/biomedical/disease/disease_chart.test.tsx

```diff
@@ -175,3 +176,26 @@ test("getCompoundDiseaseContraindication", () => {
     }
   }
 });
+
+test("getDiseaseCommonName", () => {
```

Comment:
it looks like this file is testing the functions in the file data_processing_utils.ts, so this test file should be named data_processing_utils.test.ts

For a file named abc.ts, the corresponding test file should be named abc.test.ts

Reply:
Thank you for pointing out the file naming rules. I changed the name to `data_processing_utils.test.ts` for both the protein and disease directories.